language/tools/move-cli/tests/testsuite/republish_cyclic/args.txt

```diff
@@ -0,0 +1,6 @@
+publish src1 -v
+view storage/0x00000000000000000000000000000042/modules/M.mv
+view storage/0x00000000000000000000000000000043/modules/N.mv
+publish src2 -v
```

Comment:
Not needed for this PR, but in a follow-up would be great to:
- add cyclic dep checks in `move doctor`
- show that after `publish src2` fails`, `publish src2 --ignore-breaking-changes` succeeds + running `move doctor` after detects the problem.

Reply:
I'll add a commit to this PR for this (otherwise I might forgot to create a new PR...)

Reply:
Updated